lib/honeybadger/cli/test.rb

```diff
@@ -162,7 +162,7 @@ def verify_test
         if calling = TestBackend.callings[:notices].find {|c| c[0].exception.eql?(TEST_EXCEPTION) }
           notice, response = *calling
 
-          if response.code != 201
+          if !response.success?
```

Comment:
FYI `unless response.success?` would have [been better](https://github.com/bbatsov/ruby-style-guide#unless-for-negatives) in this case.

Reply:
Good catch. I think I may refactor this to remove the nested conditional too.